Resources/views/CRUD/base_show.html.twig

```diff
@@ -11,6 +11,14 @@ file that was distributed with this source code.
 
 {% extends base_template %}
 
+{% block title %}
+    {{ "title_show"|trans({'%name%': admin.toString(object)|truncate(15) }, 'SonataAdminBundle') }}
```

Comment:
do we need to use `admin.translationDomain` instead of SonataAdminBundle directly? Not sure how this is done on the other cases.